tools/project_manager/page_compare.php

```diff
@@ -0,0 +1,182 @@
+<?php
+$relPath="./../../pinc/";
+include_once($relPath.'base.inc');
+include_once($relPath.'misc.inc');
+include_once($relPath.'theme.inc');
+include_once($relPath.'Project.inc');
+include_once($relPath."unformat.inc"); // remove_formatting(), unspace()
+
+require_login();
+
+$projectid = validate_projectID('project', @$_GET['project']);
+
+$project = new Project($projectid );
+
+$state = $project->state;
+$title = $project->nameofwork;
+$sub_title = _('Compare pages with formatting removed');
+
+output_header( "$sub_title: $title", NO_STATSBAR);
+
+echo "<h1>$title</h1>\n";
+echo "<h2>$sub_title</h2>\n";
+
+$project_url = "$code_url/project.php?id=$projectid&amp;expected_state=$state";
+$label = _("Return to Project Page");
+
+echo "<p><a href='$project_url'>$label</a></p>\n";
+
+if (!$project->pages_table_exists)
+{
+    echo "<p>";
+    if ($project->archived != 0) {
+        echo _("The project has been archived, so page details are not available.");
+    } elseif ($project->state == PROJ_DELETE) {
+        echo _("The project has been deleted, so page details are not available.");
+    } else {
+        echo _("Page details are not available for this project.");
+    }
+    echo "</p>";
+    exit();
+}
+
+$L_round_id = array_get($_GET, "L_round_id", "P3");
+$R_round_id = array_get($_GET, "R_round_id", "F1");
+$page_set = array_get($_GET, "page_set", "all");
+$round_selector = new RoundSelector($page_set);
+$round_selector->render($projectid, $L_round_id, $R_round_id);
+
+// if this is first entry don't do anything else
+if(!isset($_GET['compare']))
+    exit();
+
+$state_index = array_flip($PROJECT_STATES_IN_ORDER);
+
+if($state_index[$project->state] < $state_index["{$R_round_id}.proj_avail"])
+{
+    echo "$R_round_id has not started";
+    exit;
+}
+
+$L_round = get_Round_for_round_id($L_round_id);
+$R_round = get_Round_for_round_id($R_round_id);
+$L_text_column_name = $L_round->text_column_name;
+$R_text_column_name = $R_round->text_column_name;
+$L_round_num = $L_round->round_number;
+$R_round_num = $R_round->round_number;
+
+$right_complete = ($state_index[$project->state] >= $state_index["{$R_round_id}.proj_done"]);
+$username = $pguser;
+
+switch($page_set) {
+    case 'right':
+        $condition = "$R_round->user_column_name = '$username'";
+        break;
+    case 'left':
+        $condition = "$L_round->user_column_name = '$username'";
+        break;
+    default: // all
+        $condition = "1";
+        break;
+}
+
+if(!$right_complete)
+    $condition .= " AND state='$R_round->page_save_state'";
+
+$query = "
+SELECT image, $L_text_column_name, $R_text_column_name
```

Comment:
Please indent these 4 spaces so it stands out as SQL. Also, move the closing quote to a line by itself. See Project:get_hold_status() in `pinc/Project.inc` for an example.